src/classes/LGEncryption.ts

```diff
@@ -137,4 +137,14 @@ export class LGEncryption {
 
     return Buffer.concat([ivEnc, dataEnc]);
   }
+
+  decrypt(cipher: Buffer) {
```

Comment:
Can you add tests for this function to `test/LGEncryption.test.ts` with sample payloads?